### PR TITLE
Fix %!s(<nil>) in  doc gen

### PR DESCRIPTION
**Pull request by sneaxiy**
Change the doc of `unknownType` parameter from 
```
<td>max_bin</td>
<td>%!s(<nil>)</td>
<td>Only used if tree_method is set to hist, Maximum number of discrete bins to bucket continuous features.</td>
```
to be
```
<td>max_bin</td>
<td>unknown</td>
<td>Only used if tree_method is set to hist, Maximum number of discrete bins to bucket continuous features.</td>
```

### CODE REVIEW

File: pkg/attribute/attribute.go [hunk 1]
```diff
@@ -321,8 +321,14 @@ func (d Dictionary) GenerateTableInHTML() string {
 	<td>%s</td>
 	<td>%s</td>
 </tr>`
+
+		var typ interface{} = desc.typ
+		if typ == nil {
+			typ = "undetermined"
```
Comment (shendiaomo) [inline pkg/attribute/attribute.go]: Should "undetermined" be consistent with "unknown" in the codebase?

File: go.mod [hunk 2]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
 	github.com/beltran/gohive v1.0.0 // indirect
+	github.com/bitly/go-simplejson v0.5.0
```
Comment (weiguoz) [inline go.mod:13]: Why we need the `go-simplejson` here?

Comment (sneaxiy) [reply go.mod:13]: I do not know why. But the linter did add these.

File: pkg/attribute/attribute.go (hunk 1 above)
Comment (sneaxiy) [reply pkg/attribute/attribute.go]: Done.

**Review by shendiaomo — APPROVED** [review]
lgtm

**Pull request merged**